tests/test_get_keepbits.py

```diff
@@ -29,3 +30,42 @@ def test_get_keepbits_inflevel_dim(rasm_info_per_bit, inflevel):
     if isinstance(inflevel, (int, float)):
         inflevel = [inflevel]
     assert (keepbits.inflevel == inflevel).all()
+
+
+def test_get_keepbits_informationFilter():
+    ds = xr.tutorial.load_dataset("air_temperature")
+    info = xb.get_bitinformation(ds, dim="lat")
+    var = info["air"]
+    for i in range(var.size):
+        if i >= 19 and i <= 24:
+            var[i] = 0.05
```

Comment:
Do you intend to add artificial information here? Could you just add a comment above these lines describing what you are doing here?

Reply:
Yes I have added information from my side here, Is that fine?

Reply:
Can you push your edits. I currently can't see any added comments.